shared/CliContext.cs

```diff
@@ -8,14 +8,13 @@ namespace Microsoft.Extensions.Tools.Internal
     public static class CliContext
     {
         /// <summary>
-        /// dotnet --verbose subcommand
+        /// dotnet -d|--diagnostics subcommand
```

Comment:
Was this just wrong before?

Reply:
This was `dotnet --verbose` was renamed after the project.json -> msbuild conversion. Just updating the doc comment to be accurate.